tests/unit/Scaffold/Modes/AppendModeTest.php

```
@@ -109,6 +110,61 @@ public function testResultHashMatchesActualFileHash(): void
+    {
+        $sourcePath = "{$this->tempDir}/provider/stubs/source.txt";
+
+        mkdir(dirname($sourcePath), 0777, recursive: true);
+        file_put_contents($sourcePath, 'x');
+
+        MockerState::addCondition(
+            'yii\\scaffold\\Scaffold\\Modes',
+            'file_get_contents',
+            [$sourcePath, false, null, 0, null],
+            false,
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xepozz\InternalMocker\MockerState' in method 'testThrowsWhenSourceReadFails'. (undefined)

(StaticAccess)